chidioguejiofor/ojumo-club-graphql
# #163295204: Initialises repo

#### What does this PR achieve?
- adds tsconfig.json
- initialises npm
- creates a basic folder structure for the app 
- initialises prisma in the api/models folder

#### Manual Testing Steps
- Clone the repo via: `git clone https://github.com/chidioguejiofor/ojumo-club-graphql.git`
- Run Start via: `npm start` 
- Some text should be printed on your console

#### Any new deployment instructions
- none

#### New environmental Variables?
- none
